dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1505,19 +1602,47 @@ def ones_like(
     return call_origin(numpy.ones_like, x1, dtype, order, subok, shape)
 
 
-def ptp(arr, axis=None, out=None, keepdims=numpy._NoValue):
+def ptp(
+    arr,
```

Comment:
to align with numpy and cupy signature
```suggestion
    a,
```